packages/integrations/node/test/trailing-slash.test.js

```diff
@@ -76,6 +76,14 @@ describe('Trailing slash', () => {
 				expect(res.status).to.equal(200);
 				expect($('h1').text()).to.equal('One');
 			});
+
+			it('Does not add trailing slash to subresource urls', async () => {
+				const res = await fetch(`http://${server.host}:${server.port}/some-base/one.css`);
+				const css = await res.text();
+
+				expect(res.status).to.equal(200);
```

Comment:
I'm not sure what's gonig here, I think your local fork is outdated and it's using the old test file. We don't use `chai` anymore. Please update your fork, and use `assert` from `node:assert/strict`

Reply:
I think the test was never updated, it was left as .js, not .test.js , I just pushed a rebase trying to avoid relying on `chai`.

Reply:
Replaced `expect` with `assert.equal` 👍🏻